api/namex/constants/__init__.py

```diff
@@ -392,7 +392,7 @@ class NameRequestActions(AbstractEnum):
     UPGRADE = 'UPGRADE'
     REAPPLY = 'REAPPLY'
     RECEIPT = 'RECEIPT'
-    COMPLETE = 'COMPLETE'
+    COMPLETE = 'CREATE'
```

Comment:
Is it worth changing the enum to CREATE as well so it doesn't confuse us later?

Reply:
Done